packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -109,6 +109,14 @@ contract Deploy is Deployer {
         vm.stopBroadcast();
     }
 
+    /// @notice Modifier that will only allow a function to be called on devnet.
+    modifier onlyDevnet() {
+        uint256 chainid = block.chainid;
+        if (chainid != Chains.LocalDevnet && chainid != Chains.GethDevnet) {
```

Comment:
Shouldn't this be `chainid == Chains.LocalDevnet || chainid == Chains.GethDevnet`?

also, btw, is `chains.OPLocalDevnet` still useful?

Reply:
Good catch, can confirm from the logs

```
Syncing deployment AddressManager: contract AddressManager
  Syncing deployment L1CrossDomainMessenger: contract L1CrossDomainMessenger
  Syncing deployment L1CrossDomainMessengerProxy: contract ResolvedDelegateProxy
  Syncing deployment L1ERC721Bridge: contract L1ERC721Bridge
  Syncing deployment L1ERC721BridgeProxy: contract Proxy
  Syncing deployment L1StandardBridge: contract L1StandardBridge
  Syncing deployment L1StandardBridgeProxy: contract L1ChugSplashProxy
  Syncing deployment L2OutputOracle: contract L2OutputOracle
  Syncing deployment L2OutputOracleProxy: contract Proxy
  Syncing deployment OptimismMintableERC20Factory: contract OptimismMintableERC20Factory
  Syncing deployment OptimismMintableERC20FactoryProxy: contract Proxy
  Syncing deployment OptimismPortal: contract OptimismPortal
  Syncing deployment OptimismPortalProxy: contract Proxy
  Syncing deployment ProxyAdmin: contract ProxyAdmin
  Syncing deployment SystemConfig: contract SystemConfig
  Syncing deployment SystemConfigProxy: contract Prox
```